main.go

```diff
@@ -88,6 +88,20 @@ func waitForDependencies() {
 			log.Println("Waiting for:", u.String())
 
 			switch u.Scheme {
+			case "file":
+				wg.Add(1)
+				go func(u url.URL) {
+					defer wg.Done()
+					ticker := time.NewTicker(waitRetryInterval)
+					var err error
+					for range ticker.C {
+						if _, err = os.Stat(u.Path); !os.IsNotExist(err) {
```

Comment:
This error check looks wrong.  I think you want `err == nil` here and then don't log an error if the err is `os.IsNotExist`.  Any other error, you would want to log.